js/index.js

```diff
@@ -40,3 +40,125 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Nav Anchors
+const navBar = document.querySelectorAll('a');
+//Attempted to use forEach to run through each 'a' tag and give it 'nav-item-x' content
+//Will come back and try for loop once MVP complete, writing them out one by one for now
+
+// navBar.forEach(navItem => navItem.textContent = siteContent['nav'][`nav-item-${navBar.length -= (navBar.length -= 1)}`]);
+
+
+navBar[0].textContent = siteContent['nav']['nav-item-1'];
+navBar[1].textContent = siteContent['nav']['nav-item-2'];
+navBar[2].textContent = siteContent['nav']['nav-item-3'];
+navBar[3].textContent = siteContent['nav']['nav-item-4'];
+navBar[4].textContent = siteContent['nav']['nav-item-5'];
+navBar[5].textContent = siteContent['nav']['nav-item-6'];
+
+navBar.forEach(element => element.style.color = 'green');
+
+const navigation = document.querySelector('nav')
+
+const newNavItemPre = document.createElement('a');
+newNavItemPre.href = '#';
+newNavItemPre.textContent = 'Prepend Item';
+newNavItemPre.style.color = 'green';
+
+const newNavItemApp = document.createElement('a');
+newNavItemApp.href = '#';
+newNavItemApp.textContent = 'Append Item';
+newNavItemApp.style.color = 'green';
+
+navigation.prepend(newNavItemPre);
+navigation.append(newNavItemApp);
+
+
+//Section CTA-----------------------------------------------
+//h1 needs line breaks
+//const lineBreak = document.createElement('br');
+
+
+//h1
+const domIsAwesome = document.querySelector('h1');
+const beforeH1 = document.createElement('h1');
+const afterH1 = document.createElement('h1');
+const newH1Array = siteContent['cta']['h1'].split(' ');
+
+
+const allCtaH1 = document.querySelectorAll('.cta-text h1');
+//console.log(allCtaH1);
+
+
+//console.log(h1Array);
+
+beforeH1.textContent = newH1Array[0];
+domIsAwesome.textContent = newH1Array[1];
+afterH1.textContent = newH1Array[2];
+
+//***note to self:APPENDS/PREPENDS MUST BE AT THE BOTTOM!!!!
+domIsAwesome.append(afterH1);
+domIsAwesome.prepend(beforeH1);
+
+//button
+const button = document.querySelector('button');
+button.textContent = siteContent['cta']['button'];
+
+//cirlce header img
+const circleImg = document.getElementById('cta-img');
+circleImg.setAttribute('src', siteContent['cta']['img-src']);
+circleImg.alt = "Header Cirlce Image";
+
+//-------------------------------------------------------------
+
+
+//MAIN CONTENT------------------------------------------------
```

Comment:
your assignment has been of the best ones in regards to organization and "human readable" so THANK YOU for that. and for the love of god keep doing this.